pitch.md

```diff
@@ -3,42 +3,65 @@
 
 ## The user and a language
 This section describes who the project would serve and why a language might be a
 good way to meet their needs.
 
+The user would be game developers (or perhaps artists).
 
 ### What's the need?
 _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+Well, existing map generators that I'm aware of generally have set tile sizes, which limits the ability to quickly design maps that don't look as blocky or aren't constrained by having every piece of the map fill a tile of the same size. But to have complete control over a map would require illustrating it entirely from scratch, which would take too long. This would be a tool that might keep the process quick while allowing for more artistic design decisions. 
 
 ### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+If existing map makers count as DSLs, then these tools have already proven to address this type of need.
+Perhaps a text-based DSL isn't the most ideal way to go, but it wouldn't be a long process to make a map, and it would be easy to make minor changes without having to handle multiple layers in a software like photoshop.
 
 ### Why you?
 _What excites you about this idea? How did you come up with it?_
 
+_Words cannot express my interest in game design._
+Also, map generation is one of the most basic game dev tools, and free-to-use tools can always be improved on.
 
 ### Interface (syntax)
 _How might the user interact with the language? What does programming look 
 like? Why is this the right way to interact with the problem domain? Be careful
 to distinguish between, e.g,. a graphical interface and a linguistic interface._ 
 
+This is mostly described above. A program is just lines of code. The specifics haven't been worked out yet, but specifying tile objects might look something like:
+
+tile water1 = SRCPATH
+tile ground1 = SRCPATH {
+	edge = SRCPATH
+}
+freeform tile peninsula = SRCPATH {
+	anchor = (5,5)
+}
+
 
 ### Operation (semantics)
 _What might happen when a program runs? How does a program interact with the
 user? What kinds of semantic (i.e., non-syntax) errors might occur, and how 
 might they be communicated to the user?_
 
+When the program runs, it generates an image file.
+One possibility, to help communicate with the users, is to also output a map with the outlines of each area/tile and anchors marked.
```

Comment:
This seems really useful. If two areas (desert and beach?) looked similar it might be hard to tell if your map is correct, but if it also outputted the same map but without inserting the actual images, that would be useful. That would also be useful if someone wanted to design a map before they had the actual drawings.